src/OrleansAzureUtils/OrleansAzureUtils.csproj

```diff
@@ -114,25 +114,11 @@
     <None Include="project.json" />
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <!-- Begin Orleans: Without these lines the project won't build properly -->
-  <PropertyGroup>
-    <OrleansProjectType>Server</OrleansProjectType>
```

Comment:
these particular lines aren't needed, though - we don't use this property anymore